src/database/models/repository.py

```diff
@@ -4,10 +4,12 @@
 
 from database.manager import base_model
 from sqlalchemy import Column, Integer, String, DateTime, Boolean
+from sqlalchemy.orm import relationship
 
 
 # ORM Class for Repositories
 class Repository(base_model()):
+    from database.models.language import repo_language_table
```

Comment:
You should be able to import this outside of the class scope - you only reference repository in a string instead of directly - so you shouldn't run into any circular import issues.